pkg/cmd/openshift-apiserver/openshiftapiserver/images.go

```diff
@@ -0,0 +1,35 @@
+package openshiftapiserver
+
+import (
+	"fmt"
+
+	"github.com/golang/glog"
+
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	kapi "k8s.io/kubernetes/pkg/apis/core"
+	coreinternalinformer "k8s.io/kubernetes/pkg/client/informers/informers_generated/internalversion/core/internalversion"
```

Comment:
seems like low hanging fruit to switch to external?

Reply:
> seems like low hanging fruit to switch to external?

probably, but not here.